examples/noisy_label_detection/trak_noisy_label.py

```diff
@@ -1,36 +1,43 @@
 """This example shows how to use the IF to detect noisy labels in the MNIST."""
 
+import argparse
+
 import torch
 from torch import nn
 from torchvision import datasets, transforms
 
 from dattri.algorithm.trak import TRAKAttributor
-from dattri.benchmark.datasets.mnist import train_mnist_lr
+from dattri.benchmark.datasets.cifar2 import train_cifar2_resnet9
 from dattri.benchmark.utils import SubsetSampler, flip_label
+from dattri.metric import mislabel_detection_auc
 from dattri.task import AttributionTask
 
 
-def get_mnist_indices_and_adjust_labels(dataset):
-    dataset.targets, flip_index = flip_label(dataset.targets, p=0.1)
+def get_cifar_indices_and_adjust_labels(dataset, subset_indice):
+    dataset.targets, flip_index = flip_label(
+        torch.tensor(dataset.targets)[subset_indice], p=0.1
+    )
     return flip_index
 
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--device", type=str, default="cuda")
+    args = parser.parse_args()
+
+    # create cifar 10 dataset
```

Comment:
cifar 10 or cifar 2?

Reply:
it's cifar10 because we find that cifar2 noisy label detection is far more difficult than cifar10. That's actually an interesting observation.